Fix comment and spec parameter inaccuracies (#11)
alchemyplatform/EIPs: EIPS/eip-6900.md

#### Calls to `installPlugin`

> The function `installPlugin` accepts a parameters for the address of the plugin to install, the Keccak-256 hash of the plugin's manifest, ABI-encoded data to pass to the plugin's `onInstall` callback, and an array of addresses that represent the plugin's install dependencies.

Reviewer: ```suggestion
The function `installPlugin` accepts parameters for the address of the plugin to install, the Keccak-256 hash of the plugin's manifest, ABI-encoded data to pass to the plugin's `onInstall` callback, and an array of addresses that represent the plugin's install dependencies.
```

Reviewer 2: ```suggestion
The function `installPlugin` accepts 4 parameters: the address of the plugin to install, the Keccak-256 hash of the plugin's manifest, ABI-encoded data to pass to the plugin's `onInstall` callback, and an array of addresses that represent the plugin's install dependencies.
```

- Each execution selector MUST be added as a valid execution function on the modular account. If the execution selector has already been added or matches the selector of a native function, the function SHOULD revert.
- If an associated function that is to be added already exists, execution SHOULD NOT revert but continue to the next operation.

> Finally, the function MUST call the plugin's `onInstall` callback with the data provided in the `data` parameter. This serves to initializes the plugin state for the modular account. If `onInstall` reverts, the `installPlugin` function MUST revert.

Reviewer: ```suggestion
Finally, the function MUST call the plugin's `onInstall` callback with the data provided in the `installData` parameter. This serves to initializes the plugin state for the modular account. If `onInstall` reverts, the `installPlugin` function MUST revert.
```

#### Calls to `uninstallPlugin`

> The function `uninstallPlugin` accepts a parameters for the address of the plugin to uninstall and ABI-encoded data to pass to the plugin's `onUninstall` callback.

Reviewer: ```suggestion
The function `uninstallPlugin` accepts parameters for the address of the plugin to uninstall and ABI-encoded data to pass to the plugin's `onUninstall` callback.
```

Reviewer 2: ```suggestion
The function `uninstallPlugin` accepts 2 parameters: the address of the plugin to uninstall and ABI-encoded data to pass to the plugin's `onUninstall` callback.
```

The function MUST parse through the execution functions, validation functions, and hooks in the manifest and remove them from the modular account after resolving each `ManifestFunction` type. If any resolved function to be removed no longer exists on the modular account, execution SHOULD NOT revert to allow the uninstall to complete.

> Finally, the function MUST call the plugin's `onUninstall` callback with the data provided in the `data` parameter. This serves to clear the plugin state for the modular account. If `onUninstall` reverts, execution SHOULD continue to allow the uninstall to complete.

Reviewer: ```suggestion
Finally, the function MUST call the plugin's `onUninstall` callback with the data provided in the `uninstallData` parameter. This serves to clear the plugin state for the modular account. If `onUninstall` reverts, execution SHOULD continue to allow the uninstall to complete.
```
